read.g

```diff
@@ -1,26 +1,31 @@
-DeclareInfoClass("InfoGauss");
-SetInfoLevel(InfoGauss, 0);
+#
+# GaussPar: Parallel implementation of gaussian algorithm for finite fields.
+#
+# Reading the implementation part of the package.
+#
 
+# The gauss pkg doesn't work under old versions of HPCGAP. But we can take
+# the functions we need from the following files we copied.
 gauss := LoadPackage("GAUSS");
 if gauss = fail then
-    Read("./hpc/gauss-upwards.gd");
-    Read("./hpc/gauss-upwards.gi");
+    ReadPackage( "GaussPar", "gap/hpc/gauss-upwards.gd");
```

Comment:
What's the difference between `Read` and `ReadPackage( "GaussPar", "..." )`?

Reply:
Good question. That's the whole purpose of turning it into a package. :) As long as our package lives in a folder where GAP looks for packages GAP will know the path to our package. Then all files can be read with the `ReadPackage` command, no matter what the current working directory is.

Since we used relative path names with `Read`, the user needed to start GAP in the root directory of our repository.